external_distributions/mumps_and_scalapack/scalapack_python3_scripts/README

```diff
@@ -0,0 +1,25 @@
+Upgraded python scripts in scalapack installer to python3, using 2to3 and
+some additional tweaks applied with plenty of hand-holding from Puneet.
+When scalapack is being built these modified files overwrite the ones
+contained in the scalapack tar file.
+
+  Careful: scalapack.log is also scanned to extract some build information
+ for the subsequent mumps installation, particularly information about
+ mangling, so adding debug statements to the python scripts can have
+ unintended consequences (as I found out).
+
+  To compare the modified files against the original ones:
+
+    cd external_distributions/mumps_and_scalapack/scalapack_python3_scripts/
+    mkdir junk
+    cp scalapack_installer.tgz junk/
+    cd junk/
+    tar xvfz scalapack_installer.tgz 
+    cd scalapack_installer_0.94/script/
+    find . -name '*.py' -exec meld {} ../../../scalapack_python3_scripts/{} \;
```

Comment:
Assumes people have `meld` -- I don't. (IIRC, there's no support for it on macOS.) [As an alternative, you can replace `meld` with `git diff --no-index --`](https://stackoverflow.com/a/15110387).

Reply:
Done (though I'm not convinced git is quite the replacement; suggested using diff or sdiff).